include/hpcombi/bmat8_impl.hpp

```diff
@@ -36,70 +36,71 @@ static const constexpr std::array<uint64_t, 8> COL_MASK = {
     0x1010101010101010, 0x808080808080808,  0x404040404040404,
     0x202020202020202,  0x101010101010101};
 
-static const constexpr std::array<uint64_t, 64> BIT_MASK = {{0x8000000000000000,
```

Comment:
This change seems unrelated to the changes to the documentation, probably it will disappear if the files are clang-formatted.

Reply:
As you want. I won't go into any debate here. I just would like to mention that to my eye such a big hole in the code makes it harder to understand the structure of the code.